.github/workflows/python-app.yml

```diff
@@ -25,7 +25,7 @@ jobs:
         python -m pip install --upgrade pip
         pip install flake8 pytest
         # install dependencies. Eventually, this should all be in in requirements.txt 
-        pip install astrodbkit2
+        pip install git+https://github.com/dr-rodriguez/AstrodbKit2
```

Comment:
I dont understand why we arent using the pip version?

Reply:
By changing to the github version we can capture the latest work, as opposed to released versions